src/shared/components/NcTile/NcTile.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="tile" :class="{active: localeActive}" @click="$emit('set-template')">
+	<div :tabindex="tabbable ? 0 : null" class="tile" :class="{active: localeActive}" @click="$emit('set-template')">
```

Comment:
For screen readers https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected would probably be the best option here to have a grid role on the parent element and then indicate the selected state through aria so that it is read out.

Reply:
Not blocking as this could be a follow up, but was also mentioned in #248